### ISSUE TITLE
Minishift did not start - Checking container network ... ^@FAIL

### ISSUE THREAD

**Issue by quiel77** [minishift/minishift#870]
Hi Guys,
I am trying to run the Minishift on my MAC and it throws me this error:
```
_-- Checking container networking ... ^@FAIL
   Error: containers cannot communicate with the OpenShift master
   Details:
     The cluster was started. However, the container networking test failed.
   Solution:
     Ensure that access to ports tcp/8443 and udp/53 is allowed on 192.168.64.2.
     You may need to open these ports on your machine's firewall.
   Caused By:
     Error: Docker run error rc=1
     Details:
       Image: openshift/origin:v1.5.0
       Entrypoint: [/bin/bash]
       Command: [-c echo 'Testing connectivity to master API' && curl -s -S -k https://192.168.64.2:8443 && echo 'Testing connectivity to master DNS server' && for i in {1..10}; do    if curl -s -S -k https://kubernetes.default.svc.cluster.local; then       exit 0;   fi;    sleep 1; done; exit 1]_
```

Anyone faced this before ?
Thanks,
Eze

Note (gbraad): *code formatted*

**Comment by budhrg** [comment]
@quiel77 , could you add debug logs with `minishift start -v 5 --show-libmachine-logs` ?

**Comment by hferentschik** [comment]
Do you have any sort of firewalls or similar in place? Is this problem reproducible? If you do a _minishift delete_ and start over, does it always fail the network check? If so, you can test from your host whether the VM IP is reachable. You should also check whether the OpenShit pod still runs:

    $ eval $(minishift docker-env)
    $ docker ps

You should be seeing a _openshift/origin_ running. If not, OpenShift might have started, but unexpectedly quit. If so, it might help to look at the logs of the exited container. List all containers and determine the id of the exited OpenShift container:

    $ docker ps -a
    ...
   34d4dee98f52        openshift/origin:v1.5.0                   "/bin/bash -c '#/bin/"   18 minutes ago      Exited (0) 17 minutes ago

   $ docker logs 34d4dee98f52

**Comment by quiel77** [comment]
Hi @budhrg @hferentschik  I deleted the minishift, update the brew and installed minishift again. Now works but with the VirtualBox Driver. It is failing with xhyve, 
start.go:171] Error starting the VM: Unused explicit driver options:map[xhyve-uuid:F4BB3F79-AB4E-4708-95CA-E32FBFCDEFD3]. Retrying.

**Comment by praveenkumar** [comment]
@quiel77 can you delete your minishift using `minishift delete` and then restart your mac then try using xhyve?

**Comment by quiel77** [comment]
I will try it
Thanks 

Enviado desde mi iPhone

**Comment by hferentschik** [comment]
Closing this. User got it to work and otherwise also no more info.

**Comment by dhanugithub** [comment]
Hello,
I am running Minishift on windows 7 with virtualbox.

Minishift VM starts and openshift origin works fine but it seems docker is not working properly.

$minishift docker-env
SET DOCKER_TLS_VERIFY=1
SET DOCKER_HOST=tcp://192.168.99.101:2376
SET DOCKER_CERT_PATH=C:\Users\dhanashree\.minishift\certs
REM Run this command to configure your shell:
REM     @FOR /f "tokens=*" %i IN ('minishift docker-env') DO @call %i

$docker ps
The command "docker" is either misspelled or
could not be found.

I get this output. How to I check if networking is setup correctly? And how to use boot2docker ISO that is there in minishift?
Thank you.

**Comment by dhanugithub** [comment]
Hello,
I am running Minishift on windows 7 with virtualbox.

Minishift VM starts and openshift origin works fine but it seems docker is not working properly.

$minishift docker-env
SET DOCKER_TLS_VERIFY=1
SET DOCKER_HOST=tcp://192.168.99.101:2376
SET DOCKER_CERT_PATH=C:\Users\dhanashree\.minishift\certs
REM Run this command to configure your shell:
REM     @FOR /f "tokens=*" %i IN ('minishift docker-env') DO @call %i

$docker ps
The command "docker" is either misspelled or
could not be found.

I get this output. How to I check if networking is setup correctly? And how to use boot2docker ISO that is there in minishift?
Thank you.

**Comment by praveenkumar** [comment]
> I get this output. How to I check if networking is setup correctly? 

@dhanugithub this more looks like you didn't have docker client installed on windows, first you need to install it to make it work.

**Comment by dhanugithub** [comment]
Thank you @praveenkumar for quick reply. I just downloaded and installed Docker Toolbox (following https://docs.docker.com/toolbox/toolbox_install_windows/). It created a VM called 'default' in virtualbox.
Before installing docker client, I stopped my minishift VM, now when I restarted it to check if docker works or not, the VM started with different IP address and gave error. How should I proceed now?
Thank you
